packages.yml

```diff
@@ -0,0 +1,6 @@
+packages:
+  - package: fivetran/fivetran_utils
+    version: [">=0.4.0", "<0.5.0"]
+
```

Comment:
Do we use fivetran_utils at all in this repo? If not, we should consider removing it to reduce the dependency tree.